notebooks/huggingface_notebook.ipynb

```diff
@@ -0,0 +1,296 @@
+{
```

Comment:
Can you also add this to the notebook table: https://docs.mosaicml.com/en/stable/getting_started/notebooks.html